src/DynamicRupture/FrictionLaws/FrictionLaws.h

```diff
@@ -1,6 +1,8 @@
 #ifndef SEISSOL_FRICTIONLAWS_H
 #define SEISSOL_FRICTIONLAWS_H
 
+// IWYU pragma: begin_exports
```

Comment:
Does this PR silently add IWYU? Do we enforce it somewhere?

Reply:
It's enabled since clang-tidy 17. So yeah, we basically run it now as well... :)

Reply:
I tried IWYU once, but I didn't like that they always create forward definitions, if possible. Yeah it's more efficient, but also kind of annoying.